lib/src/rendering/rich_editor.dart

```diff
@@ -54,8 +55,7 @@ class RichEditorState extends State<RichEditor> {
 
   void _handleRequest(HttpRequest request) {
     try {
-      if (request.method == 'GET' &&
-          request.uri.queryParameters['query'] == "getRawTeXHTML") {
+      if (request.method == 'GET' && request.uri.queryParameters['query'] == "getRawTeXHTML") {
```

Comment:
please reformat the file with dartFmt. i think this line is longer than it should be